src/server/game/Globals/ConversationDataStore.cpp

```diff
@@ -106,7 +107,8 @@ void ConversationDataStore::LoadConversationTemplates()
             uint32 conversationId         = fields[0].GetUInt32();
             uint32 actorId                = fields[1].GetUInt32();
             ObjectGuid::LowType actorGuid = fields[2].GetUInt64();
-            uint16 idx                    = fields[3].GetUInt16();
+            uint32 actorNearId            = fields[3].GetUInt32();
+            uint16 idx = fields[4].GetUInt16();
```

Comment:
indent

Reply:
Yes, preferably keep the codestyle from the lines above.